### ISSUE TITLE
Incorrect render of html blockcode

### ISSUE THREAD

**Issue by SharkProgramming** [Requarks/wiki#1521]
**Describe the bug**
There is a problem with the render of a complete html page in blockcode

**To Reproduce**
1. Create a page 
2. Insert blockcode with html source page like:
```html
<!DOCTYPE html>
<html>
<head>
<title>Error</title>
<style>
    body {
        width: 35em;
        margin: 0 auto;
        font-family: Tahoma, Verdana, Arial, sans-serif;
    }
</style>
</head>
<body>
<h1>An error occurred.</h1>
<p>300</p>
</body>
</html>
```
3. Save the page and you can see that the </body> is going outside the blockcode
4. Take a look at the table of content , it appears a link called An error occured

**Expected behavior**
All html code have to stay inside the blockcode in the page visible by every user of the wiki.

**Screenshots**
![issue_render_html](https://user-images.githubusercontent.com/33226400/75155037-693db880-570f-11ea-8f98-52aaba62fdef.png)

**Host Info (please complete the following information):**
 - OS: Linux - Ubuntu Linux (bionic) 18.04 x64
 - Wiki.js version: 2.1.113
 - Database engine: PostgreSQL 10.10

**Additional**
The preview in markdown editor looks good

**Comment by chris-m92** [comment]
I've had that too. A workaround I've used is to use `&lt;` and `&gt;` to escape standard tags like `h1` and `div`.

**Comment by NGPixel** [comment]
This was fixed in 2.2. please upgrade to the latest version.
